index.html

```diff
@@ -26,14 +26,15 @@
 <body>
   <nav class="destop-version">
       <a href="#" class="my-logo">Alexander</a>
-    <nav class="hamburger-menu">
+    <div class="open-menu">
       <a href="#"><i class="fa fa-bars"></i></a>
-    </nav>
+    </div>
         <ul class="nav-links hidden right-nav">
-          <li><a href="#portfolio">Portfolio</a></li>
-          <li><a href="#About">About</a></li>
-          <li><a href="#Contact">Contact</a></li>
+          <li><a href="#portfolio">Portfolio <span><i class="fas fa-chevron-right first"></i></span></a></li>
+          <li><a href="#About" class="about">About<span><i class="fas fa-chevron-right second"></i></span></a></li>
+          <li><a href="#Contact">Contact<span><i class="fas fa-chevron-right last"></i></span></a></li>
```

Comment:
- [ ] In the below pic, When the user click (or tap) any option of the mobile menu, the mobile menu should disappear. 

![Screenshot (1377)](https://user-images.githubusercontent.com/22434660/123002243-53a5d500-d3cf-11eb-8fcc-519c981f6cbf.png)